### ISSUE TITLE
Code line replicating error

### ISSUE THREAD

**Issue by htdbui** [julianfaraway/LMP#2]
https://github.com/julianfaraway/LMP/blame/589a78976c12ebe2305a01e6692ab798948838ea/pyscripts/estim.py#L200

The code on line 200 is the same as the one on the line 189. But the code on line 200 should calculate the covariance of paramaters, whereas the code on the line 189 calculate the covariance of predictors.

**Comment by julianfaraway** [comment]
Yes - this should read `lmod.cov_params()`. Now corrected - thanks
